src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -6511,13 +6566,15 @@ void StorageReplicatedMergeTree::movePartitionToTable(const StoragePtr & dest_ta
         dest_table_storage->waitForAllReplicasToProcessLogEntry(entry);
     }
 
-    Coordination::Requests ops_dest;
+    Coordination::Requests ops_src;
 
-    ops_dest.emplace_back(zkutil::makeCreateRequest(
+    ops_src.emplace_back(zkutil::makeCreateRequest(
         zookeeper_path + "/log/log-", entry_delete.toString(), zkutil::CreateMode::PersistentSequential));
-    ops_dest.emplace_back(zkutil::makeSetRequest(zookeeper_path + "/log", "", -1));  /// Just update version
+    ops_src.emplace_back(zkutil::makeSetRequest(zookeeper_path + "/log", "", -1));  /// Just update version
```

Comment:
Maybe add to comment: `because merges assignment relies on it` or something like this.